src/main/java/seedu/address/model/person/Status.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a Person's status in the address book.
+ * Guarantees: immutable;
+ */
+public class Status {
+    private enum HealthStatus { GREEN, YELLOW, RED };
+    private final String status;
+
+    public Status(String status) {
+        this.status = status;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    @Override
+    public String toString() {
+        return this.status;
+    }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
When overriding equals(), you need to override hashCode() as well to prevent unexpected behaviours